packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1037,10 +1037,51 @@ class PopupMenuButton<T> extends StatefulWidget {
   final bool captureInheritedThemes;
 
   @override
-  _PopupMenuButtonState<T> createState() => _PopupMenuButtonState<T>();
+  PopupMenuButtonState<T> createState() => PopupMenuButtonState<T>();
 }
 
-class _PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
+/// The [State] for a [PopupMenuButton].
+///
+/// See [showButtonMenu] for a way to programmatically open the popup menu
+/// of your button state.
+class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
+  /// A method to show a popup menu with the items supplied to
+  /// [PopupMenuButton.itemBuilder] at the position of your [PopupMenuButton].
+  ///
+  /// By default, it is called when the user taps the button and [PopupMenuButton.enabled]
+  /// is set to `true`.
+  /// Moreover, you can open the button by calling the method manually.
```

Comment:
would you remove the new line between line1052 and line1053?

Reply:
There are two trailing spaces in line 1052, which will push the next line to a new line in Markdown. Do you think that it is not appropriate?